tests/deposit_withdraw_test.ts

```diff
@@ -4,12 +4,20 @@ import { withL1Client, L1Client } from "../src/clients/client";
 import { getConfigByChainName } from "zkwasm-deployment/src/config";
 import { L1ClientRole } from "zkwasm-deployment/src/types";
 import { test_verify } from "./test_utils";
+import { encodeL1address } from "web3subscriber/src/addresses";
 
-const initial_root: Uint8Array = new Uint8Array([166, 157, 178, 62, 35, 83, 140, 56, 9, 235, 134, 184, 20, 145, 63, 43, 245, 186, 75, 233, 43, 42, 187, 217, 104, 152, 219, 89, 125, 199, 161, 9]);
+const deposit_root: Uint8Array = new Uint8Array([166, 157, 178, 62, 35, 83, 140, 56, 9, 235, 134, 184, 20, 145, 63, 43, 245, 186, 75, 233, 43, 42, 187, 217, 104, 152, 219, 89, 125, 199, 161, 9]);
```

Comment:
Please change back to initial root. This is only for intial (empty 20 depth m tree), not for deposit.